.azure-pipelines/templates/maven-preview/copy-and-build.yml

```diff
@@ -0,0 +1,18 @@
+steps:
+- task: CopyFiles@2
+  inputs:
+    SourceFolder: '$(Agent.TempDirectory)'
+    Contents: '**'
+    TargetFolder: '$(DROP_PATH)/'
+
+- task: Gradle@2
+  inputs:
+    gradleWrapperFile: '$(DROP_PATH)/gradlew'
+    workingDirectory: '$(DROP_PATH)/'
+    options: '-x jar -x javadoc -x javadocJar -x compileJava -x processResources -x classes -x sourceJar'
+    tasks: 'publishSnapshotPublicationToSonatypeSnapshotRepository'
```

Comment:
set this as a template parameter and merge it with the other copy and build file